components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/UserSessionStore.java

```diff
@@ -975,7 +1094,7 @@ public boolean hasExistingFederatedAuthSession(String idpSessionIndex) throws Us
      * @return True if a federated auth session found with the given session index.
      * @throws UserSessionException If an error occurred while checking for an federated auth session.
      */
-    public boolean isExistingFederatedAuthSessionAvailable(String idpSessionIndex, int tenantId)
+    public boolean hasExistingFederatedAuthSession(String idpSessionIndex, int tenantId)
```

Comment:
why have we changed a public method name here?

Reply:
Reverted to back to the previous name with [1c921fe](https://github.com/wso2/carbon-identity-framework/pull/4558/commits/1c921fee7a66cc83e784b863c8fae6a4e13e902e)